call-home/src/bin/stats/main.rs

```diff
@@ -0,0 +1,104 @@
+use clap::Parser;
+mod cache;
+mod store;
+use crate::{
+    cache::{events_cache::{Cache, EventSet}},
+    store::events_store::initialize,
+};
+use obs::common::errors;
+use k8s_openapi::api::core::v1::ConfigMap;
+use mbus_api::mbus_nats::{message_bus_init, NatsMessageBus};
+use std::time::Duration;
+use tracing::{error, info};
+use utils::{
+    raw_version_str,
+    tracing_telemetry::{default_tracing_tags, flush_traces, init_tracing},
+};
+
+pub(crate) async fn mbus_init(mbus_url: &str) -> errors::Result<NatsMessageBus> {
+    let message_bus: NatsMessageBus = message_bus_init(mbus_url).await;
```

Comment:
IMHO  NatsMessageBus is pub and we can use it

Reply:
The problem is bus crate, it's exposing internal details about the bus, that is a specific nats message bus struct!
The mbus crate should expose the Bus trait here. 
If otherwise the interface is correct, you can probably use the trait here, instead of NatsMessageBus